fastcdc/fastcdc.h

```diff
@@ -239,7 +239,8 @@ class ChunkerTmpl {
       len = cfg_.max_size;
     }
 
-    uint64_t hash = 0;
+    // Init hash to all 1's to avoid zero-length chunks with min_size=0.
+    uint64_t hash = (uint64_t)-1;
```

Comment:
This option SGTM. In practice, we always choose min_size > 0, anyway, so it's a rather theoretical consideration where we should just make sure that things don't explode.

Reply:
Agreed.